src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -3,10 +3,16 @@
 import re
 import time
 import zipfile
+from os import PathLike
+from pathlib import Path
 import httpx
 from bs4 import BeautifulSoup
+from bs4 import NavigableString
+from bs4 import Tag
 from deprecated import deprecated
 from progress.bar import Bar
```

Comment:
```suggestion
from nplinker.genomics.antismash import \
    download_and_extract_antismash_data
```

Reply:
Why should we delete deprecated and Bar libraries imports?

Reply:
Miswritten :-)